dev/tests/integration/testsuite/Magento/MediaGallery/Model/ResourceModel/AssetKeywordsTest.php

```diff
@@ -66,45 +66,57 @@ protected function setUp(): void
      *
      * @magentoDataFixture Magento/MediaGallery/_files/media_asset.php
      * @dataProvider keywordsProvider
-     * @param array $keywords
+     * @param array|null $keywords
+     * @param array|null $updatedKeywords
```

Comment:
These parameters cannot be null, also detailed array type declaration is welcome)
```suggestion
     * @param string[] $updatedKeywords
```